tests/composite_pk/test_get.py

```diff
@@ -124,3 +126,10 @@ def test_lookup_errors(self):
 
     def test_get_user_by_comments(self):
         self.assertEqual(User.objects.get(comments=self.comment_1), self.user_1)
+
+    def test_get_next_by_field(self):
+        self.assertEqual(self.stamp_1.get_next_by_created(), self.stamp_2)
+        self.stamp_1.delete()
+        msg = "get_next/get_previous cannot be used on unsaved objects."
+        with self.assertRaisesMessage(ValueError, msg):
+            self.stamp_1.get_next_by_created()
```

Comment:
```suggestion
        self.assertEqual(self.stamp_1.get_next_by_created(), self.stamp_2)
        msg = "get_next/get_previous cannot be used on unsaved objects."
        with self.assertRaisesMessage(ValueError, msg):
            TimeStamped(id=3).get_next_by_created()
```

I would avoid deleting something created in `setUpTestData`.
Instead, test it on unsaved object.

Reply:
Good idea - pushed some tweaks :+1:

Reply:
Your version is even better 👍